src/main/java/org/example/basics/sumOf TwoNumber.java

```diff
@@ -0,0 +1,15 @@
+class Main {
```

Comment:
- Package info missing
```java
package org.example.basics;
```

- File name should be _SumOfTwoNumber.java_ (CamelCase)